athina/evals/function/wrapper.py

```diff
@@ -0,0 +1,25 @@
+
+from typing import Optional
+from athina.evals.function.function_evaluator import FunctionEvaluator
+
+
+class ContainsAny(FunctionEvaluator):
+    def __init__(
+        self,
+        keywords: dict,
+        case_sensitive: Optional[bool] = False,
+    ):
+        super().__init__(
+            function_name="ContainsAny",
```

Comment:
Nit: would be better to grab these from the enums in `athina/evals/eval_type.py`

Reply:
Fixed